### ISSUE TITLE
Reproducing imagenet 1K from imagenet 21K pretraining

### ISSUE THREAD

**Issue by shairoz-deci** [google-research/vision_transformer#168]
I am trying to train ViT_base_16 on imagenet 1K (224x224) from an imagenet21K pretrained weights. 
Following the training recipe from the paper (batch=512, 20K steps, 500 warmup steps, lr=0.01, cosine scheduler and grad_clip of 1, dropout = 0) reaches an accuracy of 83.7 where the reported accuracy of the best available model is 84.4  

I've tried reproducing the results with several repos, lr behaves as expected yet results are not reproduced.
Can you proved code for imagenet 1K fine tunning or elaborate on additional methods used to fine tune (e.g., data augmentations)

Thanks

**Comment by CharlesLeeeee** [comment]
@shairoz-deci  do you use mixed precision training?
